TreasureHunt/Assets/2.Scripts/Hunter/FieldView/TreasureSetupController.cs

```diff
@@ -0,0 +1,137 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using SimpleJSON;
+
+public class TreasureSetupController : MonoBehaviour {
+
+	public static TreasureSetupController instance = null;
+
+	void Awake()
+	{
+		if (instance == null)
+
```

Comment:
There's a few unnecessary blank lines here. Also, you sometimes put braces around conditional (if) statements and sometimes don't. Try to be consistent.